seleniumJSEnv/src/test/scala/org/scalajs/jsenv/selenium/KeepAliveTest.scala

```diff
@@ -24,7 +22,7 @@ class KeepAliveTest {
     def quit(): Unit = closed = true
 
     def executeScript(code: String, args: Object*): Object =
-      Map.empty[String, String].asJava
+      new java.util.HashMap[String, java.util.List[String]]()
```

Comment:
Hum, why did the type of values change from `String` to `java.util.List[String]`?

Reply:
Was a bug :) But it's of course not distinguishable for an empty map...